en/middleware/microdds.md

```diff
@@ -0,0 +1,74 @@
+# MicroDDS
+
+Micro XRCE-DDS is a software solution that enables communication with an existing DDS network. 
+
+The new approach provides a faster and simpler method of integrating applictaions running and linked in DDS domains (including ROS nodes), making it easy to share sensor data, commands, and other vehicle information.
+
+The following guide describes the new PX4 bridge architecture, and shows the reader how to write a simple *Micro DDS* application to subscribe * publish telemetry updates from the PX4 Autopilot.
+
+:::note
```

Comment:
@akshata-01 FYI I switched this around so this note about fast RTPS is at the bottom. The top should all be interesting info about microDDS. I haven't rewritten that.